src/renderers/testing/ReactTestRenderer.js

```diff
@@ -76,7 +81,9 @@ class ReactTestComponent {
     transaction: ReactTestReconcileTransaction,
     context: Object,
   ) {
+    var options = transaction.getTestOptions();
     this._currentElement = nextElement;
+    this._hostNode = options.createNodeMock(nextElement);
```

Comment:
Okay. 😞 

Can we use `this._hostContainerInfo` for this instead? It's a thing that DOM renderer uses, for example, for DOM nesting validation. Both composite and host instances already store it so it's always available. It is the fourth argument to `ReactReconciler.mountComponent()` and I think it's currently `null` in the test renderer.